lib/arctic/vendor/dispersal_api.rb

```diff
@@ -76,6 +76,17 @@ def collect_order_line(shop_id, order_id, order_line)
           end
         end
 
+        # Confirm order line shipments by ids
+        #
+        # @param [String] shop_id the shop ID
+        # @param [Array<String>] order_line_ids the order line ids
+        # @return [Faraday::Response]
+        def confirm_order_line_shipments(shop_id, order_line_ids:)
+          request(:post, "shops/#{shop_id}/order_lines/confirm_shipments", body: { ids: order_line_ids }).tap do |response|
+            raise InvalidResponse, response.body unless response.status == 201
```

Comment:
you have `head :no_content` on Core which is 204 status, and here you check 201 which is `:created`